src/Raku/ast/statementprefixes.rakumod

```diff
@@ -617,8 +617,56 @@ class RakuAST::StatementPrefix::Phaser::Block
 # The FIRST phaser.
 class RakuAST::StatementPrefix::Phaser::First
   is RakuAST::StatementPrefix::Phaser::Block
+  is RakuAST::BeginTime
 {
     method type() { "FIRST" }
+
+    method is-begin-performed-before-children { True }
+    method is-begin-performed-after-children  { False }
+
+    method PERFORM-BEGIN(Resolver $resolver, RakuAST::IMPL::QASTContext $context) {
+        my $blorst := nqp::getattr(self, RakuAST::StatementPrefix, '$!blorst');
+
+        my $True := RakuAST::Term::Name.new(RakuAST::Name.from-identifier('True'));
```

Comment:
This should be an implicit lookup. Alternatively you could just put True in a RakuAST::Declaration::External::Constant. True and False are available in the bootstrap.